kobo/apps/audit_log/views.py

```diff
@@ -672,9 +733,33 @@ def list(self, request, *args, **kwargs):
 
 
 @extend_schema(
-    tags=['access-logs'],
+    tags=['Access-Logs'],
+)
+@extend_schema_view(
+    list=extend_schema(
+        description=read_md('audit_log', 'access_logs/exports/list'),
+        request=None,
+        responses=open_api_200_ok_response(AccessLogExportListInlineSerializer),
+    ),
+    create=extend_schema(
+        description=read_md('audit_log', 'access_logs/exports/create'),
+        request=None,
+        responses=open_api_202_accepted_response(AccessLogExportCreateInlineSerializer),
+    ),
 )
 class AllAccessLogsExportViewSet(BaseAccessLogsExportViewSet):
+    """
+    ViewSet for managing every user's access logs export
```

Comment:
```suggestion
    ViewSet for managing exports of every users' access logs
```

Otherwise it makes it seem like we're listing everyone's exports rather than everyone's access logs